test/5.parallel.test.js

```diff
@@ -50,7 +50,7 @@ describe('Parallel', () => {
 
     it('parallelizes execution', function(){
       this.timeout(70);
-      const actual = parallel(2, [
+      const actual = parallel(5, [
```

Comment:
This change is mostly unrelated. It corrects an earlier mistake which caused the tests to fail sometimes in Travis.